### DISCUSSION TITLE
Does anyone have this problem where on wasm, resizing can mess things up?

### DISCUSSION THREAD

**Discussion by aldelaro5** [AvaloniaUI/Avalonia#11171] [Q&A]
Basically, I can't tell if it's avalonia or wasm limitations, but any kind of resize to the browser window can cause weird things to happen.

Even things like you know the bottom bar on chrome that appears once you download something? That causes the viewport of the app to be reiszed, but you when dismiss the download bar, the area that it was occluding is now...white and it has a tendency to stay there.

There's plenty of other things too that happens on both chrome and firefox. Like if I pop the dev tools and EVER SLIGHTLY move the splitter between the dev tools and the viewport, the viewport size just craps out. It could just stay stuck and occlude a large portion of the app. Obviously, same deal for resizing the window browser itself.

My question is more why this happens? Is it Avalonia's problem? A browser's problem? It's fine IF you never resize, but it's just weird.

**Comment by timunie** [comment]
Related? #11161

**Reply by aldelaro5** [reply]
Not too sure, this seems to affect only some controls, but what I had feels a bit more global than that because resizing on desktop has no problems, it is a specific wasm thing.

I'll try to get more visuals later.
